src/main/resources/lib/credentials/select/select.css

```diff
@@ -5,19 +5,45 @@ div#credentialsDialog {overflow-y: scroll;}
 body.masked {overflow-y: hidden;}
 div.include-user-credentials { padding: 0.2em; }
 /* adapted from Jenkins' style.css */
-.help-btn { float: right; }
 .help-content {
-    border: solid #bbb 1px;
-    background-color: #f0f0f0;
-    padding: 1em;
-    margin-bottom: 1em;
+    position: relative;
+    padding: 1rem;
+    margin: 1rem 0;
     word-break: break-word;
+    border-radius: 6px;
+    z-index: 0;
 }
+
+.help-content::before {
```

Comment:
What is that used for? Can't we rely on `jenkins-help-button` or `l:helpIcon` from core?

Reply:
It’s used when you add a credential parameter to a job.

credentials plugins has created a ‘simple help’ that can have inline text. Can maybe refactor it to use core in some way, this is just fixing it up to look right.